prime-dotnet-webapi/Models/AccessTerms/LicenseClassClauseMapping.cs

```diff
@@ -0,0 +1,24 @@
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+    [Table("LicenseClassClauseMapping")]
+    public class LicenseClassClauseMapping : BaseAuditable
+    {
+        public int LicenseCode { get; set; }
+
+        [JsonIgnore]
+        public License License { get; set; }
+
+        public int OrganizatonTypeCode { get; set; }
+
+        [JsonIgnore]
+        public OrganizationType OrganizationType { get; set; }
+
+        public int LicenseClassClauseId { get; set; }
+
+        [JsonIgnore]
+        public LicenseClassClause LicenseClassClause { get; set; }
```

Comment:
The mapping should map to a licence class clause type rather than an Id so that we can find the newest one

Reply:
I thought this table just stored the newest one

Reply:
@itchison There has been a lot of back and forth with how to implement this, what do you think?

Reply:
This code is 100% going to change later. I'm cool if this works now and we can change it to suit the business later and target a better technical approach too.